src/main/java/com/internet/shop/controllers/order/CancelingOrderController.java

```diff
@@ -0,0 +1,27 @@
+package com.internet.shop.controllers.order;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.service.OrderService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/order/canceling")
+public class CancelingOrderController extends HttpServlet {
```

Comment:
the user can't cancel the order after it is completed
let's remove this controller